Teapot/MockTeapot.swift

```diff
@@ -22,38 +20,56 @@ open class MockTeapot: Teapot {
         case invalidMockFile(String)
     }
 
-    /// The status code the URL request should return
+    private let currentBundle: Bundle
+    private let resource: String
     private let statusCode: StatusCode
 
+    private var overrideEndpointDictionary = [String: String]()
```

Comment:
This sounds like a function (starting with a verb), but it's a private property. Can we rename it to make it clearer?